db/migrate/20240620121939_add_nhs_england_provider.rb

```diff
@@ -0,0 +1,5 @@
+class AddNhsEnglandProvider < ActiveRecord::Migration[6.1]
+  def change
+     Zprovider.create(zproviderid: 'X26') unless Zprovider.exists?('X26')
```

Comment:
Does this need a `shortdesc`?

Reply:
Oh, I see `shortdesc` isn't included in other migrations

Reply:
None of the past Zprovider additions have added 'shortdesc' , I suppose as its mainly staging database which finally ends up in CAS having proper Zprovider table.